File: src/main/java/by/forward/forward_system/core/dto/messenger/OrderParticipantDto.java
package by.forward.forward_system.core.dto.messenger;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class OrderParticipantDto {
    private Long id;
    private Long orderId;
    private Long userId;
    private String participantsType;
    private String participantsTypeRus;
}

File: forward-system-frontend/src/core/dto/OrderParticipantDto.ts
import type {UserDto} from "@/core/dto/UserDto.ts";
import type {ParticipantType} from "@/core/type/ParticipantType.ts";

export interface OrderParticipantDto {
    id: number,
    orderId: number,
    fee: number,
    user: UserDto,
    type: ParticipantType
    typeRusName: string
}